package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "island.is",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Was this accidental?

Reply:
yes, sorry will clean up. I think the Changelog experiments have committed locally something.